src/main/java/se/knowit/bookitregistration/controller/RegistrationController.java

```diff
@@ -51,7 +51,7 @@ public RegistrationController(RegistrationService service) {
 
     @DeleteMapping("/{id}")
     public ResponseEntity<String> deleteByRegistrationId(@PathVariable String id) {
```

Comment:
It is not required to use PathVariable annotation when the name of the variable matches the name in the PathMapping annotation. Spring figures it out.